end_to_end_tests/golden-record/my_test_api_client/models/a_discriminated_union_type_1.py

```diff
@@ -1,58 +0,0 @@
-from typing import Any, Dict, List, Type, TypeVar
-
-from attrs import define as _attrs_define
-from attrs import field as _attrs_field
-
-T = TypeVar("T", bound="ADiscriminatedUnionType1")
-
-
-@_attrs_define
-class ADiscriminatedUnionType1:
```

Comment:
Here's an example of one kind of testing I'm trying to get away from. The "golden record"-based tests work by taking a huge spec document, generating a client from it, and then verifying that the resulting code looks _exactly_ like this reference version. Since the reference version was created by the exact same process, this doesn't really verify that the code is what we want— just that the code is the same as it was before, so this really only worked as a regression test.